cmd/cstor-pool-mgmt/volumereplica/volumereplica.go

```diff
@@ -39,22 +39,18 @@ func CheckValidVolumeReplica(cVR *apis.CStorVolumeReplica) error {
 	var err error
 	if len(cVR.Labels["cstorvolume.openebs.io/uid"]) == 0 {
 		err = fmt.Errorf("Volume Name/UID cannot be empty")
-		glog.Errorf(err.Error())
```

Comment:
Is the error being logged inside the caller?

Reply:
yes